<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1489</width>
    <height>928</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AcadEref</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="txtLog">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>680</y>
     <width>651</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="altino_img">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>431</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Caslon Pro</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Caslon Pro</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1930</x>
     <y>1360</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>전방 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="robot">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>230</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="person">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>510</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtPerson">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>450</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>750</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>760</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>PORT :</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdF3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(80, 80, 80);
border-bottom-color: rgb(81, 81, 81);
border-left-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-30</y>
     <width>1531</width>
     <height>961</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>제작 : 전민규, 박민제, 정진우, 정지윤</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>알티노   자율주행 자동차 프로젝트</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>561</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>691</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeconnect">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>810</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>알티노 해제</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixmap_disconnect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>810</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pixmap_connect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>750</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>750</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>알티노 연결</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixmap_start">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>750</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pixmap_stop">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>810</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoStop">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>810</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>자동운전 종료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoStart">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>750</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>자동운전 시작</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>810</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>연결상태 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>810</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>미연결</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIRSet">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>860</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>적외선센서 초기화</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>전방 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>좌전방 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>우전방 :</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdF2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(80, 80, 80);
border-bottom-color: rgb(81, 81, 81);
border-left-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdF1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(80, 80, 80);
border-bottom-color: rgb(81, 81, 81);
border-left-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdR4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(80, 80, 80);
border-bottom-color: rgb(81, 81, 81);
border-left-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>우측 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>좌측 :</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdL5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(80, 80, 80);
border-bottom-color: rgb(81, 81, 81);
border-left-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdB6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>660</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(80, 80, 80);
border-bottom-color: rgb(81, 81, 81);
border-left-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtAltinoStatus_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>후방 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>691</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>885</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>TIME   :</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>885</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>0초</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtRobotQueView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>880</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixmap_help">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>870</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtRobot">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>200</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtRobotQue">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>220</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>안녕하세요 알티노 봇입니다</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtPersonAnw">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>480</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;</string>
   </property>
   <property name="text">
    <string>안녕?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>880</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNametxt">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>880</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>880</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userPhonetxt">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>880</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hancom Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_12</zorder>
  <zorder>label_3</zorder>
  <zorder>txtPort</zorder>
  <zorder>label_13</zorder>
  <zorder>label_2</zorder>
  <zorder>label_logo</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_16</zorder>
  <zorder>altino_img</zorder>
  <zorder>btnDeconnect</zorder>
  <zorder>pixmap_disconnect</zorder>
  <zorder>pixmap_connect</zorder>
  <zorder>btnConnect</zorder>
  <zorder>pixmap_start</zorder>
  <zorder>pixmap_stop</zorder>
  <zorder>btnAutoStop</zorder>
  <zorder>btnAutoStart</zorder>
  <zorder>label_21</zorder>
  <zorder>txtAltinoStatus</zorder>
  <zorder>btnIRSet</zorder>
  <zorder>txtAltinoStatus_2</zorder>
  <zorder>lcdF3</zorder>
  <zorder>txtAltinoStatus_3</zorder>
  <zorder>txtAltinoStatus_4</zorder>
  <zorder>lcdF2</zorder>
  <zorder>lcdF1</zorder>
  <zorder>lcdR4</zorder>
  <zorder>txtAltinoStatus_5</zorder>
  <zorder>txtAltinoStatus_6</zorder>
  <zorder>lcdL5</zorder>
  <zorder>lcdB6</zorder>
  <zorder>txtAltinoStatus_7</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>timer</zorder>
  <zorder>txtLog</zorder>
  <zorder>txtRobotQueView</zorder>
  <zorder>pixmap_help</zorder>
  <zorder>person</zorder>
  <zorder>txtPerson</zorder>
  <zorder>robot</zorder>
  <zorder>txtRobot</zorder>
  <zorder>txtRobotQue</zorder>
  <zorder>txtPersonAnw</zorder>
  <zorder>label_24</zorder>
  <zorder>userNametxt</zorder>
  <zorder>label_25</zorder>
  <zorder>userPhonetxt</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
